packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -199,6 +199,9 @@ export class FocusZone extends React.Component<IFocusZoneProps> implements IFocu
     // Dispose all events.
     this._disposables.forEach(d => d());
 
+    // Clear function references.
```

Comment:
Might be good to be even more specific:
```suggestion
    // Clear function references so their closures can be garbage-collected.
```